packages/amcacheparser.vm/amcacheparser.vm.nuspec

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>amcacheparser.vm</id>
+    <version>1.5.1.20231208</version>
+    <authors>Eric Zimmerman</authors>
+    <description>Amcache.hve parser with lots of extra features. Handles locked files</description>
+    <dependencies>
+      <dependency id="common.vm" />
+      <dependency id="dotnet-6.0-runtime" version="[6.0.25]" />
```

Comment:
I think we should introduce a dotnet-6.vm package (see https://github.com/mandiant/VM-Packages/issues/673#issuecomment-1859977796) to ensure we don't require several version of the same tool and to avoid constant dependency updates with our test suite. We should use this package in all the tools that require dotnet version 6 (the ones in this PR and [sfextract.vm](https://github.com/mandiant/VM-Packages/blob/e4cf21e70e9a8dce58449e74f49184910232648c/packages/sfextract.vm/sfextract.vm.nuspec#L10))

Reply:
#783 require dotnet 6 too

Reply:
Done in #812